frontend/src/components/Dashboard/CrossingListPage/CrossingListItem/CrossingListItem.js

```diff
@@ -39,7 +141,8 @@ class CrossingListItem extends React.Component {
       durationId: (this.state.selectedStatus === statusConstants.LONGTERM ? this.state.selectedDuration : null),
```

Comment:
Not a huge issue. Can you remove the ternaries on line 140 and 141 to close out issue #76 . Same as below, I have this in my open PR, but I don't want to merge nightmare for you.